metadata_service/proxy/atlas_proxy.py

```diff
@@ -177,6 +176,22 @@ def _parse_bookmark_qn(self, bookmark_qn: str) -> Dict:
         result = pattern.match(bookmark_qn)
         return result.groupdict() if result else dict()
 
+    def _get_user_detail(self, user_id: str, fallback: str = None) -> Union[Dict, str]:
+        """
+        Helper function to help get the user details if the `USER_DETAIL_METHOD` is configured,
+        else uses the user_id for both email and user_id properties.
+        :param user_id: The Unique user id of a user entity
+        :return: a dictionary of user details
+        """
+        if app.config.get('USER_DETAIL_METHOD'):
+            user_details = app.config.get('USER_DETAIL_METHOD')(user_id)  # type: ignore
+        elif fallback:
+            user_details = fallback
+        else:
+            user_details = {'email': user_id, 'user_id': user_id}
```

Comment:
why do we need to put user_id under email ?

Reply:
because of the validation rules and not utilizing the user_id as the PK for other proxies yet, we must have at least email in place. and for atlas we are using user_id as the CTA key.